static/header.less

```diff
@@ -112,7 +111,7 @@
 .top-right-links {
     display: none;
     @media screen and (min-width: @breakpoint){
-        display: flex;
```

Comment:
Was this supposed to be removed?

Reply:
Yes, it's now applied by the `.flex` mixin instead.